src/components/bookslist.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Book from './book';
+
+const BooksList = (props) => {
+  const { store } = props;
+  const books = store.getState();
+  return (books.map(book => (
+    <Book key={book.id} book={book} />
+  )));
+};
```

Comment:
To access the data in store we need to use the methods react provides. let us using combine and mapStateToProps to get the data from the store and pass it into the component instead of passing the whole store as props to the component. This defeats the purpose or redux.

![Screenshot 2019-12-02 at 10 28 45 AM](https://user-images.githubusercontent.com/10922508/69947788-ac143980-14ee-11ea-8954-e42616e16cab.png)